docs/_sass/_main.scss

```diff
@@ -182,6 +183,190 @@ button {
     align-content: center;
 }
 
+.logo {
+    width: 80%;
+}
+
+.search-icon {
+    margin: auto 0px;
+}
+
+.flex {
+    display: -webkit-box;
+    display: -moz-box;
+    display: -ms-flexbox;
+    display: -moz-flex;
+    display: -webkit-flex;
+    display: flex;
+    -webkit-flex-flow: row wrap;
+    flex-flow: row wrap;
+    align-content: space-between;
+}
+
+#sidebar-search {
+    background-color: $color-green-appBG;
+    width: 400px;
```

Comment:
Where does 400 come from? Again, I think we use 375px on NewDot for the right hand sidepane. 

Also, what does this look like on mobile?

Reply:
Yess, It's fixed now